xml/core_registers.xml

```diff
@@ -22,7 +32,15 @@
             When 1, {\tt ebreak} instructions in User/Application Mode enter
             Debug Mode.
         </field>
-        <field name="0" bits="11" access="R" reset="0" />
+        <field name="intdisable" bits="11" access="R/W" reset="0">
+            0: Interrupts are handled as usual.
+
+            1: Interrupts are disabled. This bit supersedes mstatus.ie.
+
+            This feature may not exist in implementations from early in the
```

Comment:
No, and this bit doesn't enable interrupts during debug mode. During debug mode interrupts are still always disabled. This bit can be used to disable interrupts while running regular code (not in debug mode), whether that be while single stepping, or running freely.

Clearly I need to work on the description, but I'm not sure how. In my reading of the text it doesn't mention anything about changing interrupt behavior during Debug Mode. What can I add to make it more clear?